rust/worker/src/log/log.rs

```diff
@@ -0,0 +1,165 @@
+use crate::chroma_proto;
+use crate::chroma_proto::log_service_client::LogServiceClient;
+use crate::config::Configurable;
+use crate::config::WorkerConfig;
+use crate::errors::ChromaError;
+use crate::errors::ErrorCodes;
+use crate::log::config::LogConfig;
+use crate::types::EmbeddingRecord;
+use crate::types::EmbeddingRecordConversionError;
+use async_trait::async_trait;
+use thiserror::Error;
+
+pub(crate) struct CollectionInfo {
+    pub(crate) collection_id: String,
+    pub(crate) first_log_id: i64,
```

Comment:
what is first_log_id - not very clear from reading the code.

Reply:
This is to match the proto buf's definition. I will add some comments to the field definition.

Reply:
cool thanks